Massive.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Arrays;
+
+public class Massive {
+    public static void main(String[] args) {
+        int result = 0;
+        int[] array = new int[]{4, 6, 8, 4};
+        System.out.println(Arrays.toString(array));
+
```

Comment:
A new method should be created according to the home task

Reply:
Example of implementation
void devider(int numbers[], int deviderValue) {
int devidedResults[] = new int[numbers.length];
try {
for (int i = 0; i < numbers.length; i++) {
int result = numbers[i] / deviderValue;
devidedResults[i] = result;
}
System.out.println(Arrays.toString(devidedResults));
} catch (ArithmeticException exception) {
System.out.println("Back to school and learn math");
}
}